ui/apps/ui/src/app/components/results-with-pagination/result.component.ts

```diff
@@ -276,6 +276,16 @@ export class ResultComponent implements OnInit {
             '<em>' + tag.split(':', 2)[1].trim() + '</em>'
           );
         }
+        if (this.highlightsreal['tag_list_tg'] === undefined) {
```

Comment:
Recommendation instead of the if else statement:
```
this.highlightsreal['tag_list_tg'] ??= [];
this.highlightsreal['tag_list_tg'].push(
  '<em>' + tag.split(':', 2)[1].trim() + '</em>'
);
```

Reply:
tthat is for further refactor of the whole file